src/Model/GroundWaterFlow/gwf3sfr8.f90

```diff
@@ -4271,6 +4271,7 @@ subroutine sfr_calc_xs_depth(this, n, qrch, d)
       else
         derv = DZERO
       end if
+      if (derv > 10) derv = 10
```

Comment:
I don't think we want these kind of conditionals. We should look for a better approach.

Reply:
@jdhughes-usgs, Agreed.  Mainly wanted to get something to pass.  Is there a common (or more sophisticated) approach for dealing with really steep slopes (`derv`) in the search algorithm?  In some of the autotests, `derv` was spiking as high as 800K+ leading to unstable results (in some cases, negative flows).